packages/rendermime-interfaces/src/index.ts

```diff
@@ -83,6 +83,11 @@ namespace IRenderMime {
     readonly name: string;
 
     /**
+     * The name of the document model type.
+     */
+    readonly modelName: string;
```

Comment:
Sure. It was not clear to me exactly how closely these should match the interfaces given in the `DocumentRegistry`. They seem close but not exact.